src/tree_data_helpers.c

```diff
@@ -567,39 +569,28 @@ ly_store_prefix_data(const struct ly_ctx *ctx, const char *value, size_t value_l
             assert(*format_p == LY_PREF_SCHEMA_RESOLVED);
             prefixes = *prefix_data_p;
         }
+        //TODO remove
```

Comment:
I suppose all of these should be removed...

Reply:
yes